undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -371,7 +371,7 @@ undertow.application-security-domain.enable-jacc=Enable authorization using JACC
 undertow.application-security-domain.referencing-deployments=The deployments currently referencing this mapping.
 undertow.application-security-domain.setting=Settings
 
-undertow.deployment=A deployment
+undertow.deployment=Information about the web configuration for this deployment.
```

Comment:
How about "Information about the web container's handling of this deployment." ?

There are quite a few attributes here that are not about configuration.

Also, undertow.deployment.servlet and undertow.deployment.websocket have the same problem as undertow.deployment -- no useful description.